Avatar Issue Fix

Dr-Gecko opened the pull request:
Implements changes for issue #60 where the avatar url isnt displayed and instead shows a Attribute error.

Dr-Gecko committed:
Avatar Issue Fix

Implements changes for issue #60 where the avatar url isnt displayed and instead shows a Attribute error.

clubby789 commented on hackthebox/user.py:
```diff
@@ -142,6 +142,7 @@ def __init__(self, data: dict, client: "HTBClient", summary: bool = False):
         self.name = data["name"]
         self.user_owns = data["user_owns"]
         self.points = data["points"]
+        self.avatar = f'https://www.hackthebox.com{data["avatar"]}'
```
The `avatar` property is currently documented as being the relative URL of the avatar (e.g. `/storage/...`) (even if it wasn't actually implemented before). The documentation for the property should either be updated to reflect this.

Additionally, I'd like to avoid having hardcoded URLs like this as it makes testing harder. Instead, I'd prefer the `constants` file to have:

```py
URL_BASE "https://www.hackthebox.com"
API_BASE = f"{URL_BASE}/api/v4/"
```
and this line updated to use this property.

Dr-Gecko committed:
Implemented Quick Fix

Removed the hard coded URL to match the documentation of relative storage links.
